dev-infrastructure/scripts/istio.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+echo "********** ISTIO Upgrade Started **************"
+# Get the current istio and check if it match target version
+export CURRENTVERSION=$(kubectl get pods -n aks-istio-system | tail -n +2 | tail -n 1 | cut -d '-' -f 2,3,4)
+if [ "$CURRENTVERSION" == "$TARGET_VERSION" ] || [ -z "$TARGET_VERSION" ]; then
+    echo "Istio is using Target Version. Exiting script."
```

Comment:
This check is superfluous - if there are no pods running the previous version of Istio to restart, we can exit implicitly.